### ISSUE TITLE
How to install on Windows?

### ISSUE THREAD

**Issue by zacharymacleod** [kaishengtai/neuralart#23]
I really want (and, apparently, half the internet) to mess with this program and make some stuff, but there's too much of a learning curve to install it on a Windows computer... :C

**Comment by soumith** [comment]
yes, unfortunately there's no good way to install this on windows. You could maybe try it via Docker or VirtualBox

**Comment by by321** [comment]
Install Anaconda Spyder first. There're a few other packages you need to install along the way. It ain't easy but it's doable.

**Comment by patrikohlsson** [comment]
If you are running Windows 10 you can activate Bash on Ubuntu under "Programs and features" on the control panel. This will allow you to install torch and run this script. Unfortunately Bash on Win10 does not have GPU support yet - but they are considering it.
